### DISCUSSION TITLE
[Account Flagged] My Colleague's GitHub account has been flagged as `Spammy` resulting in work being stalled

### DISCUSSION THREAD

**Discussion by likhinbopanna** [community/community#125328] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hi, we're aware that we should be raising a ticket for account related issues, but we're left with no other option other than coming here to gain your attention as it has been 50+ hours with no resolution.

This has happened to my colleague's other account previously and we wonder what made GitHub to reinstate that account within 12 hours back then.

We have read `code of conduct` and yes, he cannot create another account under same email. We did go through guidelines and yes, his account has been flagged `Spammy` for (violations!?) reasons that are unknown. He did raise reinstatement request (ticket number: `2805493`, `2808389`) which does not seem to help us in any way.

This is hampering not only his work but also everyone around him as his account is hidden from public and no one in the Organization is able to review the pull requests that he have raised so far causing in delays that are unintentional.

Any help could be helpful, thanks!

Reference URL - https://github.com/orgs/community/discussions/125292

CC: @pixincreate

**Comment by heckarman** [comment]
I do not understand why GitHub just makes people wait for months these days. This account was reinstated within 12 hours of appeal as I had used alias email.

**Reply by heckarman** [reply]
@queenofcorgis, 

Apologies for tagging.

Can you do something here?

Or unless a user pays for `PRO`, account restoration is not possible given that "Standard support" is not available for the mistakes not commited by the user?

**Reply by pixincreate** [reply]
Does this mean I've to pay for `PRO`? Dude, my account is `flagged` and I cannot do anything which also restricts me from `upgrading`.

**Reply by queenofcorgis** [reply]
Hi there,

Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our Code of Conduct for more information.

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @likhinbopanna, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
